doc/source/serve/production-guide/kubernetes.md

```diff
@@ -8,9 +8,10 @@ This section should help you:
 - understand how to deploy a Ray Serve application using a [RayService].
 - understand how to monitor and update your application.
 
-The recommended way to deploy Ray Serve is on Kubernetes, providing the best of both worlds: the user experience and scalable compute of Ray Serve and operational benefits of Kubernetes.
-This also allows you to integrate with existing applications that may be running on Kubernetes.
-The recommended practice when running on Kubernetes is to use the [RayService] controller that's provided as part of [KubeRay]. The RayService controller automatically handles important production requirements such as health checking, status reporting, failure recovery, and upgrades.
+Deploying Ray Serve on Kubernetes provides the scalable compute of Ray Serve and operational benefits of Kubernetes.
+This also allows you to integrate with existing applications that may be running on Kubernetes. The recommended practice when running on Kubernetes is to use the [RayService] controller that's provided as part of [KubeRay].
+
+> NOTE: Running on Ray workloads on [Anyscale](https://www.anyscale.com/get-started) provides high-availability, high-performance autoscaling, multi-region and multi-cloud capabilities out of the box.
```

Comment:
@edoakes Anything else to list here?

```suggestion
> NOTE: Running Ray workloads on [Anyscale](https://www.anyscale.com/get-started) provides high-availability, high-performance autoscaling, multi-cloud clusters, spot instance support, and more out of the box.
```

Reply:
good catch thanks. waiting for ed to respond before resolving the thread